py/desispec/brick.py

```diff
@@ -75,28 +75,30 @@ def __init__(self, bricksize=0.5):
 
     def brickname(self, ra, dec):
         """Return string name of brick that contains (ra, dec) [degrees]
-        
+
         Args:
             ra (float) : Right Ascension in degrees
             dec (float) : Declination in degrees
-            
+
         Returns:
             brick name string
         """
         inra, indec = ra, dec
         dec = np.atleast_1d(dec)
         ra = np.atleast_1d(ra)
         irow = ((dec+90.0+self._bricksize/2)/self._bricksize).astype(int)
-        names = list()
-        for i in range(len(ra)):
-            ncol = self._ncol_per_row[irow[i]]
-            j = int(ra[i]/360 * ncol)
-            names.append(self._brickname[irow[i]][j])
+
+        ncol = self._ncol_per_row[irow]
+        jj = (ra/360.0 * ncol).astype(int)
+        names = np.empty(len(ra), dtype='U8')
```

Comment:
I don't understand how that can be possible, unless the 0.25 degree bricks actually drop some significant figures to fit into that space.  I was not aware of that "feature" and I don't particularly like it.  It implies that it is not possible, even in principle, to deduce the brick size from the brick name.